### ISSUE TITLE
Incorrect indent on selector interpolation

### ISSUE THREAD

**Issue by Cactusbone** [stylus/stlint#7]
I'm getting incorrect indent error when using selector interpolation

example from http://stylus-lang.com/docs/interpolation.html#selector-interpolation
```
mySelectors = '#foo,#bar,.baz'

{mySelectors}
  background: #000
```
